conf/base.config

```diff
@@ -20,120 +20,106 @@ process {
   maxErrors = '-1'
 
   // Process-specific resource requirements
-  withName: phix_download_db {
-    time = 4.h
+  withName: BOWTIE2_HOST_REMOVAL_BUILD {
+    cpus = { check_max (10 * task.attempt, 'cpus' ) }
+    memory = { check_max (20.GB * task.attempt, 'memory' ) }
+    time = { check_max (4.h * task.attempt, 'time' ) }
+  }
+  withName: BOWTIE2_HOST_REMOVAL_ALIGN {
+    cpus = { check_max (10 * task.attempt, 'cpus' ) }
+    memory = { check_max (10.GB * task.attempt, 'memory' ) }
+    time = { check_max (6.h * task.attempt, 'time' ) }
+  }
```

Comment:
I think this file is missing the standard process labels (process_high, process_low, ...) from the template, right?
Would be good to include (and use) them. Because all modules are build to use them.

Reply:
I agree. I am actually surprised that FASTP is not failing, since it uses `label 'process_medium' `. I had the weirdest error messages with missing labels in the base.config

Reply:
Added the labels to the `base.config` for the use with nf-core modules and to avoid errors in the future, but kept the `withName: XY` definitions for local modules.